config/defaults.mk

```diff
@@ -33,17 +34,29 @@ INSTALL = /usr/bin/install
 STATIC = YES
 SHARED = NO
 
+# CUDA configuration options
+MFEM_CUDA_CXX = nvcc
+MFEM_CUDA_FLAGS = -x=cu --expt-extended-lambda -arch=sm_60
```

Comment:
Do we want to default to sm_60 or sm_70 here?  The Volta cards supports sm_70.

Reply:
It is best to add a variable `MFEM_CUDA_ARCH` with some default that can be changed by the user.

Reply:
I believe sm_60 is the minimal with the features we need, @camierjs can confirm

Reply:
I think the best would be to compile for all arch above the minimum required.

Reply:
(The `-arch` flag is not limited to one architecture)

Reply:
> I think the best would be to compile for all arch above the minimum required.

Wouldn't that make building slow?

Reply:
I doubt it, it would be very simple to test and compare.

Reply:
The `okina-atomic` merge should let you try now by setting the `MFEM_CUDA_ARCH` to the right architecture for you (`sm_53`).